vision_evaluation/evaluators.py

```diff
@@ -126,15 +126,20 @@ def calculate_score(self, average='macro'):
             by support (the number of true instances for each label).
         ``'samples'``:
             Calculate metrics for each instance, and find their average.
+        filter_out_zero_tgt : bool
+        Removes target columns that are all zero. For precision calculations this needs to be set to False, otherwise we could be removing FP
         """
+        filter_out_zero_tgt = kwargs.get('filter_out_zero_tgt', True)
```

Comment:
calculate_score() method is not supposed to be exposed to outside. I guess we can add this argument to the method, isntead of using kwargs.